config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
```

Comment:
Given you've already opted in to the 7.0 defaults, I think we can already lose this file.